scroll-search.lic

```diff
@@ -0,0 +1,459 @@
+=begin
+  Documentation: TBD
+=end
+
+custom_require.call(%w(common common-items drinfomon))
+
+# Define Scroll Info class
+class ScrollInfo
+  def initialize(container, index, container_index, ordinal, spell, type)
+    # Instance variables
+    @container = container
+    @index = index
+	@container_index = container_index
+    @ordinal = ordinal
+    @spell = spell
+    @type = type
+  end
+
+  def get_container
+    return @container
+  end
+
+  def get_spell
+    return @spell
+  end
+
+  def get_ordinal
+    return @ordinal
+  end
+  
+  def get_index
+	return @index
+  end
+  
+  # represents the local index of the
+  # scroll within the container
+  def get_container_index
+	return @container_index
+  end
+
+  def get_type
+    return @type
+  end
+
+  def set_container(container)
+    @container = container
+  end
+  
+  def set_container_index(index)
+	@container_index = index
+  end
+  
+  def set_ordinal(ordinal)
+	@ordinal = ordinal
+  end
+end
+
+class ScrollSearch
+  include DRC
+  include DRCI
+
+  def initialize
+
+    # Variables
+    $scroll_count = 0

```

Comment:
To avoid collisions with other scripts, I recommend these be instance variables (`@`) instead of globals (`$`).